test/sanity/issue6229-mac-executeScript/test.py

```diff
@@ -0,0 +1,35 @@
+import time
+import os
+import platform
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+if platform.system() != 'Darwin':
```

Comment:
Please enable this case for all platforms, although it was reported for mac but it applies to all. And please rename the case as well.

Reply:
Updated and tested on Windows/Linux with v0.26.0 and v0.27.4, got same result as Mac OS, "issue6229-mac-executeScript" renamed to "issue6229-webview-executeScript"